diff -Nur wifi-radar-1.9.6.orig/wifi-radar wifi-radar-1.9.6/wifi-radar
--- wifi-radar-1.9.6.orig/wifi-radar	2006-03-27 23:41:40.000000000 +0100
+++ wifi-radar-1.9.6/wifi-radar	2006-07-21 14:52:19.000000000 +0100
@@ -57,10 +57,10 @@
 #
 CONF_FILE		 = "/etc/wifi-radar.conf"
 
-IWLIST_COMMAND	 = "iwlist"
-IWCONFIG_COMMAND = "iwconfig"
-IFCONFIG_COMMAND = "ifconfig"
-ROUTE_COMMAND	 = "route"
+IWLIST_COMMAND	 = "/sbin/iwlist"
+IWCONFIG_COMMAND = "/sbin/iwconfig"
+IFCONFIG_COMMAND = "/sbin/ifconfig"
+ROUTE_COMMAND	 = "/sbin/route"
 
 os.environ['LC_MESSAGES'] = 'C'
 
